### DISCUSSION TITLE
PatchTool 6.2 8/29

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#224] [Announcements]
更新漢化、修正 #222 。

檔案：
- FFXIVChnTextPatch-0829.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- rawexd-zhcn-0829.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- [backup-0823.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.67/backup-0823.rar)：乾淨的0a0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.68'>v2.68</a>.</em>

**Comment by ryanho** [comment]
檔案好像不完整，只有部份漢化？漢化執行到中途就停了，不像之前的版本會出現完成的訊息。

**Reply by GpointChen** [reply]
我的是正常的，你的停下來是停在哪裡？有log嗎？

**Reply by ryanho** [reply]
剛剛重新把漢化程式再重新解壓縮後，先執行還原再按漢化有顯示漢化完成了，進遊戲確認有完整漢化。

昨天的情況是點擊漢化按鈕後，應該是在進行更換文本的部份，進度視窗突然消失，沒有提示錯誤或者完成，進到遊戲裡頭只有很少的部份被被漢化。log檔案裡頭也沒有明顯的錯誤訊息或者是例外情況。

總之問題已經解決，感謝。

**Comment by KppKerry** [comment]
辛苦了
